swagger.yml

```
@@ -0,0 +1,137 @@
+swagger: '2.0'
+
+# This is your document metadata
+info:
+  version: "0.0.0"
+  title: Communication_Notebook
+
+# Describe your paths here
+paths:
+  # This is a path endpoint. Change it.
+  /kid/{kidId}:
```

普通は以下のようにします

```suggestion
  /kid/:id:
```

* :id のようにして、変数であることを示します
* kidのidであることは自明なので、kidは不要です
* idが複数箇所pathに含まれるような場合は、**Idと命名したほうが自然です

---

また、pathは
`/kids/:id` のほうが一般的です
REST APIについて調べてみてください